tests/accelerators/test_accelerator_connector.py

```diff
@@ -397,7 +397,14 @@ def creates_processes_externally(self) -> bool:
 
 @mock.patch.dict(
     os.environ,
-    {"SLURM_NTASKS": "2", "SLURM_JOB_NAME": "SOME_NAME", "SLURM_NODEID": "0", "LOCAL_RANK": "0", "SLURM_LOCALID": "0"},
+    {
+        "SLURM_NTASKS": "2",
+        "SLURM_JOB_NAME": "SOME_NAME",
+        "SLURM_NODEID": "0",
+        "LOCAL_RANK": "0",
+        "SLURM_PROCID": "0",
+        "SLURM_LOCALID": "0",
```

Comment:
Removing this comma will undo this unrelated change

```suggestion
        "SLURM_LOCALID": "0"
```